pkg/asset/manifests/openshift.go

```diff
@@ -44,6 +46,12 @@ type Openshift struct {
 	FileList []*asset.File
 }
 
+// Used for templating the baremetal CR.
+type BaremetalTemplateData struct {
+	Baremetal     *baremetal.Platform
+	RhcosImageURL string
+}
+
```

Comment:
This should perhaps go into pkg/types/baremetal ?

Reply:
At the very least if it remains here it shouldn't be exported (e.g. just baremetalTemplateData), it's only ever used here.

Reply:
Right good point.  Thanks guys.

Reply:
The other alternative would be to define this in pkg/types/baremetal and try to use it both here and in the bootstrap templating.